pkgs/top-level/all-packages.nix

```
@@ -11285,9 +11285,8 @@ in
 
   assimp = callPackage ../development/libraries/assimp { };
 
-  asio = asio_1_12;
   asio_1_10 = callPackage ../development/libraries/asio/1.10.nix { };
-  asio_1_12 = callPackage ../development/libraries/asio/1.12.nix { };
+  asio = callPackage ../development/libraries/asio/1.16.nix { };
```

Sorry for not requesting this earlier. Can we also do a file rename please.
```suggestion
  asio = callPackage ../development/libraries/asio/default.nix { };
```